.drone.yml

```diff
@@ -1,6 +1,13 @@
 workspace:
+  base: /srv/app
   path: src
 
+clone:
+  git:
+    image: plugins/git:latest
+    depth: 50
```

Comment:
well, if we ever wanna do `git diff --numstat master..HEAD` for speeding up tests we might want `depth: 50` 🤔